### PR TITLE
Rework response headers to have content-type and cache-control. Fixes…

### PR DESCRIPTION
**Pull request by AutomatedTester**
… #375

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/webdriver/545)

### CODE REVIEW

File: webdriver-spec.html [hunk 1]
```diff
@@ -806,8 +806,15 @@
   to the description of <var>status</var> in the <a>status code registry</a>.
 
  <li><p><a data-lt='set header'>Set</a> the <var>response</var>'s <a>header</a>
-   with <a data-lt='header name'>name</a> equal to "<code>Content-Type</code>" and
-   <a data-lt="header value">value</a> equal to "<code>application/json</code>".
+   with <a data-lt='header name'>name</a> and <a data-lt="header value">value</a>
```
Comment (shs96c) [inline webdriver-spec.html]: Nit: differing quote marks. No biggie but tweaks the OCD :)

File: webdriver-spec.html [hunk 2]
```diff
@@ -806,8 +806,15 @@
...
+   with the following values:
+   <dl>
+     <dt>"<code>Content-Type</code>"
+     <dd>"<code>application/json</code>"
```
Comment (shs96c) [inline webdriver-spec.html]: Should also specify encoding. `application/json; charset=utf-8`

File: webdriver-spec.html (hunk 1 above)
Comment (AutomatedTester) [inline webdriver-spec.html]: fixed, put more mixed quotes all over the place now

File: webdriver-spec.html (hunk 2 above)
Comment (AutomatedTester) [inline webdriver-spec.html]: added

**Comment by shs96c** [comment]
Squashed and pushed manually.

**Pull request closed without merging**